<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>671</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>待抢座列表</string>
   </property>
   <widget class="QTableView" name="seat_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>671</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="change_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>修改待抢座列表</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="user_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>291</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>抢座</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>日期：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="begin_date"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="begin_time"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>请求间隔（秒）：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="interval"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>最大请求次数：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="max_try_times"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="status">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="about_button">
       <property name="text">
        <string>关于</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="imme_run_button">
       <property name="text">
        <string>立即开始抢座</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="wait_run_button">
       <property name="text">
        <string>指定时间开始抢座</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
